src/src/FHSS.cpp

```diff
@@ -127,17 +127,17 @@ void FHSSrandomiseFHSSsequence()
             }
         }
 
-        Serial.print(FHSSsequence[i]);
-        if ((i + 1) % 10 == 0)
-        {
-            Serial.println();
-        }
-        else
-        {
-            Serial.print(" ");
-        }
+        // Serial.print(FHSSsequence[i]);
```

Comment:
I think it would be good to add build flag here to make debugging easier if something pop up with FHSS sequences.